src/tests/runTests.ts

```diff
@@ -1,26 +1,25 @@
-import * as path from 'path';
-import { runTests } from '@vscode/test-electron';
+import * as path from "path";
+import { runTests } from "@vscode/test-electron";
 
 async function main() {
     try {
         // The folder containing the Extension Manifest package.json
         // Passed to `--extensionDevelopmentPath`
-        const extensionDevelopmentPath = path.resolve(__dirname, '../../');
+        const extensionDevelopmentPath = path.resolve(__dirname, "../../..");
```

Comment:
Had to change the typescript root folder from `src` to the project root, because some `.ts` files import `package.json` from the root, and so we want the compiler to be aware of that file. That means that the `out` folder now contains `src` and everything is nested one level deeper, hence the change in the relative path here.

Note, the `out` folder is only used for running tests, not building the vsix, and tests are now running successfully.

Reply:
Tats asked the question of whether the `tsconfig` change would increase the vsix side (because it's looking at more than just the `src` folder). It doesn't increase the size because webpack only pulls in what is needed starting from `/src/extension.ts`.

However, for performance reasons it's probably best to ensure that we're not looking at unnecessary folders for TypeScript compilation, linting, and prettifying. I'll add another update for this.

Reply:
Updated. `tsconfig.json` now uses `includes` (specifying `src`) instead of `excludes`. This still avoids the error when files are pulled in from outside the `src` folder.